azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/CallingEventHandler.java

```diff
@@ -3,16 +3,14 @@
 
 package com.azure.android.communication.ui;
 
-import com.azure.android.communication.ui.configuration.events.CommunicationUIErrorEvent;
-
 /**
  * {@link CallingEventHandler}&lt;T&gt;
  *
  */
-public interface CallingEventHandler {
+public interface CallingEventHandler<T> {
     /**
-     * A callback method to process error event
-     * @param eventArgs {@link CommunicationUIErrorEvent}
+     * A callback method to process event
+     * @param eventArgs {@link T}
      */
-    void handle(CommunicationUIErrorEvent eventArgs);
+    void handle(T eventArgs);
```

Comment:
Interesting changing this to a generic. Is this considered a breaking change?

Reply:
I think this is a merge conflict. We just removed generics during release. Branch feature/remote_participant_persona_injection was created before that change merged to develop/beta.